docs/src/main/paradox/scala/http/routing-dsl/testkit.md

```diff
@@ -110,6 +110,14 @@ This is what the full working test looks like:
 @@snip [TestKitFragmentSpec.scala](../../../../../test/scala/docs/http/scaladsl/server/TestKitFragmentSpec.scala) { #source-quote }
 
 
+## Increase Timeout
+
+The default timeout when testing your routes using the testkit is 1 second. Sometimes, though, this might not be enough.
+In order to extend this default timeout, to say 5 seconds, just add the following implicit in scope:
```

Comment:
So, by default, the Scala testkit is 1 second (with dilation), when extending the timeout, one can decide to use the dilated variant or not.
However, the Java testkit, is not using any dilation and on the current implementation, it can't have it as `RouteTest` does not extend `JavaTestKit`, that is where the `dilated` method is defined.

Shall I document that the Scala testkit is using dilated by default, and that one can get advantage of it when overriding it? And then, create another issue to have `RouteTest` extend `JavaTestKit` and use `dilated` on the default timeout?